### DISCUSSION TITLE
Minimal height for State Timeline rows

### DISCUSSION THREAD

**Discussion by erwiese** [grafana/grafana#40858] [Dashboards, Panels & Visualizations]
Hi,
it would be nice to set something like a minimal row height for State Timeline rows/band.
To avoid views as you can see here where you can't read the series names:
![state-timeline-exa](https://user-images.githubusercontent.com/11369197/138668320-cfc6a41f-5260-4924-a230-d348dc85a1f3.PNG)

The number of series varies in my case, the panel height is fix, so maybe scrolling within the panel would be necessary.
A very helpful component.
Cheers

**Comment by leeoniya** [comment]
dupe of https://github.com/grafana/grafana/discussions/38303

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
